tests/config/test_config.yaml

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
Glad you did this -- I had been meaning to do it for a long time.
Is this whole file indented as well?

Reply:
This config file is for the purpose of running qc plot tests (pre-determined file names, plots at each stage, etc.). You can see `load_test_config()` as a fixture for every plot qc test function. This is in case the user uses some custom default_config that would make the tests break. Does this answer your q?